packages/backend/discovery/parallel/ethereum/diffHistory.md

```diff
@@ -1,3 +1,205 @@
+Generated with discovered.json: 0x40ee005b1b236066f02ef1c4a0f5dfd9252b45d3
+
+# Diff at Thu, 11 Apr 2024 06:24:39 GMT:
+
+- author: sekuba (<sekuba@users.noreply.github.com>)
+- comparing to: main@379d5924e19a3a6dfc1858baca3e1ce1c43bfe6f block: 19532058
+- current block number: 19630573
+
+## Description
+
+### New deploys
+
+OneStepProofEntry, OneStepProverHostIo, OneStepProverMath, OneStepProver0, OneStepProverMemory are redeployed and Reader4844 is added to to be compatible with EIP-4844.
+
+- OneStepProverHostIo: Compatibility with EIP-4844, most updates are KZG-related in funtion `executeReadPreImage()`; new function `modExp256()`
+- Reader4844.sol: New unverified contract with the functions `getBlobBaseFee()`, `getDataHashes()` for reading blob data (hashes) (info from its interface)
+
+### SequencerInbox
+
+In general most updates are EIP-4844 related, also the new role `batchPosterManager` is added and there are new checks regarding native tokens.
+
+- Batch data flag-based decoding refined for blobs: `DATA_AUTHENTICATED_FLAG`, `BROTLI_MESSAGE_HEADER_FLAG`, `DAS_MESSAGE_HEADER_FLAG`, `DATA_BLOB_HEADER_FLAG`, `TREE_DAS_MESSAGE_HEADER_FLAG`, `ZERO_HEAVY_MESSAGE_HEADER_FLAG`
```

Comment:
we probably already went through the same with arbitrum, what about checking the diff with it?

Reply:
`SequencerInbox` is the same as Arbitrum except for the timeVariation format changes. 
My comments are tightly oriented on and compared with vincenzos on the Arbitrum update